appveyor.yml

```diff
@@ -31,7 +31,7 @@
 
     environment:
         global:
-            STACK_ROOT: "c:\\sr"
+            STACK_PATH: "c:\\sr"
```

Comment:
`STACK_ROOT` is a [special environment variable for the stack tool](https://docs.haskellstack.org/en/stable/GUIDE/#setting-stack-root-location). We've set it to something short, as we have run into `MAX_PATH` issues in the past. It looks like stack and CMake are going to conflict. My proposal would be to change this to something like `STACK_ROOT_LOCATION` and use the `BOND_STACK_OPTIONS` CMake variable to set this when we invoke CMake by [adding it to `$cmakeFlags` around line 202](https://github.com/Microsoft/bond/blob/bd4b46e78a82dd3a38b52b00233a1a22047014d2/appveyor.yml#L202):

    $cmakeFlags += "-DBOND_STACK_OPTIONS=${env:STACK_ROOT_LOCATION}"